cmd/create/admin/cmd.go

```diff
@@ -222,3 +205,53 @@ func generateRandomPassword(length int) (string, error) {
 
 	return string(pw), nil
 }
+
+func FindExistingClusterAdminIDP(cluster *cmv1.Cluster, r *rosa.Runtime) (
+	htpasswdIDP *cmv1.IdentityProvider, userList *cmv1.HTPasswdUserList) {
+
+	// admin user will now be created in a htpasswd IDP named 'cluster-admin'
+	// It should suffice to look for this idp but for back-compatibility ,
+	// i.e for cases where the cluster was created  with previous versions of rosacli
+	// we are going to search  all htpasswd idps
+
+	r.Reporter.Debugf("Loading cluster's identity providers")
+	r.Reporter.Infof("Loading cluster's identity providers")
+	idps, err := r.OCMClient.GetIdentityProviders(cluster.ID())
+	if err != nil {
+		r.Reporter.Errorf("Failed to get identity providers for cluster '%s': %v", r.ClusterKey, err)
+		os.Exit(1)
+	}
+
+	for _, item := range idps {
+		if ocm.IdentityProviderType(item) == ocm.HTPasswdIDPType {
+
+			itemUserList, err := r.OCMClient.GetHTPasswdUserList(cluster.ID(), item.ID())
+			r.Reporter.Infof("user list %s: %v", item.Name(), itemUserList)
+			if err != nil {
+				r.Reporter.Errorf("Failed to get user list of the HTPasswd IDP of '%s: %s': %v", item.Name(), r.ClusterKey, err)
+				os.Exit(1)
+			}
+			if HasClusterAdmin(itemUserList) {
+				r.Reporter.Infof("Has admintrue")
```

Comment:
Is this a log we want to expose to the customer?

Reply:
nope, will fix that